lib/backend.js

```diff
@@ -48,6 +49,8 @@ function Backend(options) {
   if (!options.disableSpaceDelimitedActions) {
     this._shimAfterSubmit();
   }
+
+  logger.setLevel(options.logLevel || logger.levels.ERROR);
```

Comment:
The default is definitely set to `ERROR`. Please update the docs section so it matches.

Reply:
Good catch. I'll actually update the default to `WARN` so that we aren't swallowing too much by default.